docs/pages/reference/cli.mdx

```diff
@@ -363,6 +463,44 @@ $ tsh proxy --port 10700 app grafana &
 $ curl http://127.0.0.1:10700/api/users
 ```
 
+### tsh proxy aws
+
+Start local proxy for AWS access.  The user should already be logged into
+at least one AWS application.
+
+```code
+$ tsh proxy aws [<flags>] 
+```
+
+#### Flags
+
+| Name | Default Value(s) | Allowed Value(s) | Description |
+| - | - | - | - |
+| `--app` | Currently logged in AWS app | AWS app| Optional Name of the AWS application to use if logged into multiple |
+| `-p, --port` | none | port number | Specify the source port for the local proxy |
+| ` -e, --endpoint-url` | HTTP Proxy | Endpoint URL | Run local proxy to serve as an AWS endpoint URL. If not specified, local proxy serves as an HTTPS proxy. |
+| `-f, --format` | unix | AWS app| Optional Name of the AWS application to use if logged into multiple |
```

Comment:
Double-checking whether the final two columns of this row should be the same as those in the first row